runtime/gc_base/IdleGCManager.cpp

```diff
@@ -49,36 +49,36 @@ MM_IdleGCManager::newInstance(MM_EnvironmentBase* env)
 }
 
 void
-MM_IdleGCManager::kill(MM_EnvironmentBase* env)
+MM_IdleGCManager::kill(MM_EnvironmentBase *env)
 {
 	tearDown(env);
 	env->getForge()->free(this);
 }
 
 void
-MM_IdleGCManager::tearDown(MM_EnvironmentBase* env)
+MM_IdleGCManager::tearDown(MM_EnvironmentBase *env)
 {
-	J9HookInterface** hookInterface = _javaVM->internalVMFunctions->getVMHookInterface(_javaVM);
+	J9HookInterface **hookInterface = _javaVM->internalVMFunctions->getVMHookInterface(_javaVM);
 	if (NULL != hookInterface) {
 		(*hookInterface)->J9HookUnregister(hookInterface, J9HOOK_VM_RUNTIME_STATE_CHANGED, idleGCManagerVMStateHook, this);
 	}
 }
 
 bool
-MM_IdleGCManager::initialize(MM_EnvironmentBase* env)
+MM_IdleGCManager::initialize(MM_EnvironmentBase *env)
 {
-	J9HookInterface** hookInterface = _javaVM->internalVMFunctions->getVMHookInterface(_javaVM);
+	J9HookInterface **hookInterface = _javaVM->internalVMFunctions->getVMHookInterface(_javaVM);
 	if (NULL != hookInterface && (*hookInterface)->J9HookRegister(hookInterface, J9HOOK_VM_RUNTIME_STATE_CHANGED, idleGCManagerVMStateHook, this)) {
 		return false;
 	}
 	return true;
 }
 
 void
-MM_IdleGCManager::manageFreeHeap(J9VMThread* currentThread)
+MM_IdleGCManager::manageFreeHeap(J9VMThread *currentThread)
 {
-	MM_EnvironmentBase* env = MM_EnvironmentBase::getEnvironment(currentThread->omrVMThread);
-	MM_GCExtensions* _extensions = MM_GCExtensions::getExtensions(env);
+	MM_EnvironmentBase *env = MM_EnvironmentBase::getEnvironment(currentThread->omrVMThread);
+	MM_GCExtensions *_extensions = MM_GCExtensions::getExtensions(env);
```

Comment:
_extensions is declared as a local, so no leading '_' is needed

Reply:
one general comment that we should probably agree on.... often use env and ext as local.
we should either use them in that short form or long form (environment, extension), at least in same  method, preferably same file, ideally everywhere